station/migrations/0011_alter_station_timezone.py

```diff
@@ -0,0 +1,633 @@
+# Generated by Django 5.1 on 2024-08-19 15:38
```

Comment:
I'm not sure why this migration is included in this PR

Reply:
Because it hadn't happen before - and was necessary - so when I made the migrations actually needed for these changes, this one also popped up.